criu/mem.c

```diff
@@ -704,6 +709,8 @@ int prepare_mm_pid(struct pstree_item *i)
 			ret = collect_filemap(vma);
 		else if (vma_area_is(vma, VMA_AREA_SOCKET))
 			ret = collect_socket_map(vma);
+		else if (vma_area_is(vma, VMA_AREA_IO_URING))
+			ret = collect_io_uring_map(vma);
```

Comment:
We need to make sure that CRIU continues to work with older kernel versions that don't support io_uring. Adding a kerndat check for that might help.